HLTrigger/HLTfilters/plugins/HLTL1TSeed.cc

```diff
@@ -104,6 +107,7 @@ void HLTL1TSeed::fillDescriptions(edm::ConfigurationDescriptions& descriptions)
   desc.add<edm::InputTag>("L1JetInputTag", edm::InputTag("hltGtStage2Digis:Jet"));
   desc.add<edm::InputTag>("L1TauInputTag", edm::InputTag("hltGtStage2Digis:Tau"));
   desc.add<edm::InputTag>("L1EtSumInputTag", edm::InputTag("hltGtStage2Digis:EtSum"));
+  desc.add<edm::InputTag>("L1EtSumZdcInputTag", edm::InputTag("hltGtStage2Digis:EtSumZDC"));
```

Comment:
Let's hope L1T is using that label.

Reply:
For `L1EtSumZdcInputTag`, I kept `Zdc` as in the parameters of `L1TGlobalProducer` (https://github.com/cms-sw/cmssw/pull/42634). And yes, for `:EtSumZDC` I plan to ask that they use this name when they update `L1TRawToDigi` (unpacker).